src/components/graph/selectionToolbox/BypassButton.spec.ts

```diff
@@ -0,0 +1,114 @@
+import { mount } from '@vue/test-utils'
+import { createPinia, setActivePinia } from 'pinia'
+import PrimeVue from 'primevue/config'
+import Tooltip from 'primevue/tooltip'
+import { beforeEach, describe, expect, it, vi } from 'vitest'
+import { createI18n } from 'vue-i18n'
+
+import BypassButton from '@/components/graph/selectionToolbox/BypassButton.vue'
+import { Positionable } from '@/lib/litegraph/src/interfaces'
+import { LGraphEventMode } from '@/lib/litegraph/src/litegraph'
+import { useCommandStore } from '@/stores/commandStore'
+import { useCanvasStore } from '@/stores/graphStore'
+
+class MockLGraphNode implements Positionable {
+  type = 'TestNode'
+  title = 'Test Node'
+  mode = LGraphEventMode.ALWAYS
+  id = 'node-1'
+  pos = new Float64Array([26, 186])
+  move = vi.fn()
+  snapToGrid = vi.fn()
+  boundingRect: [number, number, number, number] = [0, 0, 100, 100]
```

Comment:
Do you need all of these properties?
Also, should the boundingRect align with pos?

Reply:
Generally, if you can, it's better to mock only what you need to be able to test the behavior that you care about.